core/tests/test_utils.py

```diff
@@ -91,6 +91,18 @@
 # We are using the b' prefix as all the stdouts are in bytes.
 LOG_LINE_PREFIX = b'LOG_INFO_TEST: '
 
+# List of model classes that don't have Takeout related class methods
+# defined because they're not used directly but only as a base classes for
+# the other models.
```

Comment:
```suggestion
# List of model classes that don't have Wipeout or Takeout, related class 
# methods defined because they're not used directly but only as 
# base classes for the other models.
```